src/opl.c

```diff
@@ -890,7 +890,15 @@ static void _loadConfig()
             configGetInt(configOPL, CONFIG_OPL_ENABLE_NOTIFICATIONS, &gEnableNotifications);
             configGetInt(configOPL, CONFIG_OPL_ENABLE_COVERART, &gEnableArt);
             configGetInt(configOPL, CONFIG_OPL_WIDESCREEN, &gWideScreen);
-            configGetInt(configOPL, CONFIG_OPL_VMODE, &gVMode);
+
+            if (!getKeyPressed(KEY_SELECT)) {
```

Comment:
according to this [list](https://www.benoitren.be/480p-palps2.html) it is common practice to hold TRIANGLE + CROSS for enabling 480p

Reply:
Also it should mentioned somewhere in the readme that such an option appeared.

Reply:
this section is actually loading the user config now and you have a condition to use auto vmode if select button is held.. im not sure this is the best spot to do it though imo

Reply:
My thought was that this would reset the Video Mode of your actual saved config - to save the user needing to hit "save changes" in the main menu when they themselves haven't actually changed any of the settings. 

It feels to me possibly confusing or at least not intuitive that you hold Triangle/Cross to reset your video mode, it works and you crack on and play some games, but then on next boot it's back at the original, incorrect (for the user) setting. 

Those were my thoughts, anyway - happy to remove the `else` if you feel otherwise :+1:

Reply:
> My thought was that this would reset the Video Mode of your actual saved config - to save the user needing to hit "save changes" in the main menu when they themselves haven't actually changed any of the settings.
> 
> It feels to me possibly confusing or at least not intuitive that you hold Triangle/Cross to reset your video mode, it works and you crack on and play some games, but then on next boot it's back at the original, incorrect (for the user) setting.
> 
> Those were my thoughts, anyway - happy to remove the `else` if you feel otherwise 👍

Makes sense I get your point but even with what you have currently it only changes the cfg in memory its not actually writing to the cfg file so a reboot with `configSetInt(configOPL, CONFIG_OPL_VMODE, gVMode);` there or not will produce the same results. If you wanted that you'd need to call `saveConfig()` at some point but doing that within `_loadConfig()` is not a good idea so probably best to just make the user save their cfg manually.

About removing the else though.. on second thought I'm pretty sure AUTO will just set it to either PAL or NTSC so you might be better off keeping the else but changing the gVMode to whatever 480p is in the enum

Reply:
Hmm, I'm convinced I had tested this logic and it had persisted across reboots, but retesting this now and yes it appears you're right. I agree calling saveConfig during a load isn't ideal. 

In terms of what we want this to reset to, I'm happy to change this to 480p if required. Is that a "safer" default to reset to than Auto? The initial issue requested 480p, but I guess we want whatever will work on the largest possible amount of displays - and we default to Auto on a default config, so I figured this would be safest. Either way it's probably worth keeping the else just for the logging?

Reply:
> Hmm, I'm convinced I had tested this logic and it had persisted across reboots, but retesting this now and yes it appears you're right. I agree calling saveConfig during a load isn't ideal.
> 
> In terms of what we want this to reset to, I'm happy to change this to 480p if required. Is that a "safer" default to reset to than Auto? The initial issue requested 480p, but I guess we want whatever will work on the largest possible amount of displays - and we default to Auto on a default config, so I figured this would be safest. Either way it's probably worth keeping the else just for the logging?

Auto is good for default value but if we look at the open issue you’re referring to the purpose of the new option is to force 480p, reason given that some newer tvs don’t support the old PAL or NTSC vmode with 480p being the lowest supported resolution I suppose..

I don’t know if this is actually the case that newer tvs don’t support it but I don’t really have any reason to doubt it and I haven’t bothered to look it up either haha